test/fixtures/controller.line/cubicInterpolationMode/scaleMaxLimit.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  config: {
+    type: 'line',
+    data: {
+      datasets: [
+        {
+          data: [
+            {x: -10, y: 150},
+            {x: 0, y: 81},
+            {x: 10, y: 49},
+            {x: 20, y: 32},
+            {x: 30, y: 21},
+            {x: 35, y: 1},
+            {x: 40, y: 16},
+            {x: 45, y: 13},
+          ],
+          borderColor: '#ff0000',
+          cubicInterpolationMode: 'monotone'
+        }
+      ]
+    },
+    options: {
+      scales: {
+        x: {display: false, type: 'linear', max: 30},
```

Comment:
Should add another test with min/max not at point. like min: 5, max: 37 with this data.

Reply:
Done!